apps/oc_chef_authz/src/oc_chef_org_user_association.erl

```diff
@@ -126,11 +127,8 @@ list(#oc_chef_org_user_association{org_id = OrgId, user_id = undefined} = Assoc,
 list(#oc_chef_org_user_association{user_id = UserId, org_id = undefined} = Assoc, CallbackFun) ->
     CallbackFun({list_query(Assoc, by_user), [UserId],  rows}).
 
-
-% Record creation via API. Note that we're using the authz_id
-% field to capture the user id, so we can use the existing framework
-% without one-offing it.
-new_record(ApiVersion, OrgId, {authz_id, UserId},  Data) ->
+new_record(ApiVersion, OrgId, unset,  Data) ->
+    UserId = eg:get({<<"user">>}, Data),
```

Comment:
As we talked about, in the slightly longer term I'd like to find a dialyzer-sanitary way to pass this through as needed, without having to place it here for temporary storage. 

Also: I don't think it's an issue in this particular case, but if you find any situation in which you're making a similar change and the ejson is being used in a PUT request, make sure to delete it from the data after retrieving - otherwise it gets sent back with the body response.